sdk/src/test/scala/com/horizen/certificatesubmitter/network/CertificateSignaturesManagerTest.scala

```diff
@@ -36,10 +37,20 @@ class CertificateSignaturesManagerTest extends JUnitSuite with MockitoSugar {
   val pubKeysNumber = 5
   val networkSettings: NetworkSettings = getMockedSettings(5 seconds)
   val params: NetworkParams = getParams(pubKeysNumber)
+  val getCertificateSignaturesSpec = new GetCertificateSignaturesSpec(pubKeysNumber)
+  val certificateSignaturesSpec = new CertificateSignaturesSpec(pubKeysNumber)
+  val messageSerializer = new MessageSerializer(Seq(getCertificateSignaturesSpec, certificateSignaturesSpec), networkSettings.magicBytes)
+
+  def roundTrip(msg: Message[_]): Message[_] = {
+    messageSerializer.deserialize(messageSerializer.serialize(msg), msg.source) match {
```

Comment:
That's not the primary purpose here, it's needed cause scorex removed the DataFromPeer message type, and now parsing is a part of Synchronizer trait => CertificateSignaturesManager cannot receive already parsed messages anymore
So this roundTrip is needed to modify the message structure as it will be done in honest communication.
Checking consistency is a side-effect here